packages/client/examples/private-geth-network.md

```diff
@@ -0,0 +1,40 @@
+# Example: Private Geth Network
+
+## Running a private network with geth
+
+In this example, we import the genesis parameters for a private Proof-of-Authority network using the geth genesis parameters format and then sync our client with a geth instance that is sealing blocks.
+
+First, create a signer account and place in the `data` directory to use with geth to seal blocks following [these instructions](https://geth.ethereum.org/docs/interface/managing-your-accounts).
+
+Next, open [these genesis parameters](./test/testdata/poa.json) and replace "728bb68502bfcd91ce4c7a692a0c0773ced5cff" with your signer address in both the `extradata` property and in the `alloc` section.
```

Comment:
noticed this typo, missing a zero, @acolytec3 this is probably where you got stuck today

```suggestion
Next, open [these genesis parameters](./test/testdata/poa.json) and replace "728bb68502bfcd91ce4c7a692a0c0773ced5cff0" with your signer address in both the `extradata` property and in the `alloc` section.
```